modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/AddressInlineFormatter.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_helper\Plugin\Field\FieldFormatter;
+
+use CommerceGuys\Addressing\Locale;
+use Drupal\address\AddressInterface;
+use Drupal\address\Plugin\Field\FieldFormatter\AddressDefaultFormatter;
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+
+/**
+ * Plugin implementation of the 'oe_theme_helper_address_inline' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "oe_theme_helper_address_inline",
+ *   label = @Translation("Inline address"),
+ *   field_types = {
+ *     "address",
+ *   },
+ * )
+ */
+class AddressInlineFormatter extends AddressDefaultFormatter implements ContainerFactoryPluginInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultSettings() {
+    return [
+      'delimiter' => ', ',
+    ] + parent::defaultSettings();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsForm(array $form, FormStateInterface $form_state) {
+    $form = parent::settingsForm($form, $form_state);
+
+    $form['delimiter'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Delimiter'),
+      '#default_value' => $this->getSetting('delimiter'),
+      '#description' => $this->t('Specify delimiter between address items.'),
+    ];
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsSummary() {
+    $settings_summary = parent::settingsSummary();
+    $example_address_items = [
+      'Rue de la Loi',
+      '56',
+      '1000',
+      'Brussels',
+    ];
+    $settings_summary[] = implode($this->getSetting('delimiter'), $example_address_items);
```

Comment:
I would remove the example and just print what the delimiter is, to have on the ui something like 
```
Delimiter: ,
```